kytos/utils/users.py

```diff
@@ -0,0 +1,144 @@
+"""Module used to handle Users in Napps Server."""
+import logging
+import re
+from getpass import getpass
+
+from kytos.utils.client import UsersClient
+from kytos.utils.config import KytosConfig
+
+log = logging.getLogger(__name__)
+
+NAME_PATTERN = ("\t- insert only letters", r'[a-zA-Z][a-zA-Z]{2,}$')
+USERNAME_PATTERN = ("\t- start with letter\n"
+                    "\t- insert only numbers and letters",
+                    r'[a-zA-Z][a-zA-Z0-9_]{2,}$')
+PASSWORD_PATTERN = ("\t- insert only the caracters:"
+                    " [letters, numbers, _, %, &, -, $]",
+                    r'[a-zA-Z0-9_%\-&$]{2,}$')
+EMAIL_PATTERN = ("\t- follow the format: <login>@<domain>\n"
+                 "\t\te.g. john@test.com", r'[^@]+@[^@]+\.[^@]+')
+PHONE_PATTERN = ("\t- insert only numbers", r'\d*$')
+
+
+class UsersManager:
+    """Class used to handle users stored by Napps server."""
+
+    attributes = {
+        "username": {"question": "Please insert your Username(Required): ",
+                     "pattern": USERNAME_PATTERN},
+        "first_name": {"question": "Please insert your First Name(Required): ",
+                       "pattern": NAME_PATTERN},
+        "last_name": {"question": "Please insert your Last Name: ",
+                      "pattern": NAME_PATTERN},
+        "password": {"question": "Please insert your Password(Required): ",
+                     "pattern": PASSWORD_PATTERN},
+        "email": {"question": "Please insert a Email(Required): ",
+                  "pattern": EMAIL_PATTERN},
+        "phone": {"question": "Please insert your Phone: ",
+                  "pattern": PHONE_PATTERN},
+        "city": {"question": "Please insert your City: ",
+                 "pattern": NAME_PATTERN},
+        "state": {"question": "Please insert your State: ",
+                  "pattern": NAME_PATTERN},
+        "country": {"question": "Please insert your Country: ",
+                    "pattern": NAME_PATTERN}
+    }
+
+    required = ["username", "first_name", "password", "email"]
+
+    def __init__(self, controller=None):
+        """If controller is not informed, the necessary paths must be.
+
+        If ``controller`` is available, NApps will be (un)loaded at runtime and
+        you don't need to inform the paths. Otherwise, you should inform the
+        required paths for the methods called.
+
+        Args:
+            controller (kytos.Controller): Controller to (un)load NApps.
+            install_path (str): Folder where NApps should be installed. If
+                None, use the controller's configuration.
+            enabled_path (str): Folder where enabled NApps are stored. If None,
+                use the controller's configuration.
+        """
+        self._controller = controller
```

Comment:
@macartur why the Controller here? This attribute (`._controller`) is not beign used anywhere. The same applies for `._config`.